### ISSUE TITLE
[3.6] Filament weight and price are not saved

### ISSUE THREAD

**Issue by Liger0** [Ultimaker/Cura#5075]
**Application Version**
3.6

**Platform**
Win 10 x64

**Printer**
Only tried 1.75 materials

**Steps to Reproduce**
Export a 1.75mm material. Delete it and re import it again

**Actual Results**
The filament cost is set to 0€ and the weight to 0g.

**Expected results**
The cost and weight should be saved in the material. 

**Additional Information**
I opened the file and there isn't any referiment to them.

**Comment by Ellecross** [comment]
Haloo @Liger0, thank you so much for reporting the issues you found. And sorry this has been like, what, 7 months too late. :|

Good thing, this is already fixed in 4.1 and is still working on the current master branch. So, I will close this as it's not an issue anymore. 

P.S. @Vandrasc and I were just talking about how we notice you report issues a lot and that's a good thing. Thank you so much for this! :D :D

**Comment by Liger0** [comment]
I don't know if the current implementation is to be considered correct.
Now, if I export a material into a file, modify the current material and rename it, then import the file I exported previously, it won't load the details I exported but the details of the material I modified which is now in effect a different material. If I unlink it from that, it resets to 0.

I also have had a weird bug when testing this. 
A material now (the silver in the image) can't be deleted even tho not used by any extruder (the material used by the extruder isn't linked to it too), and 2 materials of same brand name use 2 different dropdowns.
I tried to restart Cura with no luck.
![Materials tab](https://user-images.githubusercontent.com/20385116/60304070-8b8cf580-9938-11e9-87d5-005bada8b695.png)
[cura.zip](https://github.com/Ultimaker/Cura/files/3336785/cura.zip)
